server/prompt.go

```diff
@@ -89,43 +72,9 @@ func chatPrompt(ctx context.Context, m *Model, tokenize tokenizeFunc, opts *api.
 		prompt := msg.Content
 
 		for _, i := range msg.Images {
-			var imgData llm.ImageData
-
-			if isMllama {
-				if len(m.ProjectorPaths) == 0 {
-					imgData = llm.ImageData{
-						ID:   len(images),
-						Data: i,
-					}
-				} else {
-					data, opts, err := mllama.Preprocess(bytes.NewReader(i))
-					if err != nil {
-						return "", nil, err
-					}
-
-					buf := new(bytes.Buffer)
-					err = binary.Write(buf, binary.LittleEndian, data)
-					if err != nil {
-						return "", nil, err
-					}
-
-					ar, ok := opts["aspectRatioIndex"].(int)
-					if !ok {
-						return "", nil, fmt.Errorf("missing aspect ratio for image")
-					}
-
-					imgData = llm.ImageData{
-						ID:            len(images),
-						Data:          buf.Bytes(),
-						AspectRatioID: ar,
-					}
-				}
-				imgPrompt = "<|image|>"
```

Comment:
We were previously inserting the "<|image|>" tag here but are no longer doing it - same thing for the generate handler.

Reply:
That's not necessary anymore since these image pad tokens are now stripped out during encoding and replaced with model-specific image pad tokens.

Reply:
Do you mean in `PostTokenize`? We don't add any image tokens there for mllama. We're actually putting the image tensors on the next token, which we are assuming is the image tag.

Reply:
I've updated PostTokenize to set the token so this is no longer needed